OpenRA.Game/ModData.cs

```diff
@@ -167,6 +158,7 @@ public Map PrepareMap(string uid)
 
 			// Reinitialize all our assets
```

Comment:
Shouldn't this read 'Initialize' rather than 'Reinitialize'? As both `InitializeLoaders` and `LoadSprites` are not called anywhere else in the code base.

Reply:
No, as the comment is referring to `InitializeLoaders` which messes with static state that is reinitialized each time a new map is loaded.